frontend/src/summary.pug

```diff
@@ -23,7 +23,7 @@
       .mui-select.sort-within-group
         select(
           v-model="sortWithinGroupSelection",
-          v-bind:disabled="filterGroupSelection === 'groupByNone' || isMergeGroup"
+          v-bind:disabled="filterGroupSelection === 'groupByNone' || isAllGroupsMerged()"
```

Comment:
Its better to have a single `allGroupsMerged` instead of having both `isAllGroupsMerged` and `allGroupsMerged`, might be a little confusing.